src/Italy/Mews.Fiscalizations.Italy.Tests/UniwixClientTests.cs

```diff
@@ -129,6 +129,27 @@ private ElectronicInvoiceBody GetInvoiceBody(string invoiceNumber)
                     IssueDate = DateTime.UtcNow,
                     DocumentNumber = invoiceNumber,
                     TotalAmount = 100m
+                },
+                OrderData = new[]
+                {
+                    new OrderData
+                    {
+                        DocumentId = invoiceNumber,
+                        ItemNumber = "2",
+                        // cig
+                        TenderCode = "A1B2C3D4E5",
+                        // cup
+                        ProjectCode = "A1B2C3D4E5F6G7H"
+                    },
+                    new OrderData
+                    {
+                        DocumentId = invoiceNumber,
+                        ItemNumber = "3",
+                        // cig
+                        TenderCode = "A1B2C3D4E5",
+                        // cup
+                        ProjectCode = "A1B2C3D4E5F6G7H"
```

Comment:
Why did you add 2 items with the same data? does that bring any benefit?

Reply:
Well, I kind of assumed you have to add it for every line in your invoice.

Reply:
I would love to contrast this with some documentation but it is apparently hard to get :(